tools/fix_binary_blobs_and_parts_size_column.rb

```diff
@@ -26,5 +26,5 @@
   end
 
   # clear the binary_blob_parts from memory
-  bb.send(:clear_association_cache)
+  bb.send(:self.instance_variable_set :@association_cache, {})
```

Comment:
I think this can just be

```suggestion
  bb.reload
```